CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## Unreleased
 
+### ⚠ BREAKING CHANGES
+
+- dropped ES5 build output – this was disabled as legacy browsers are no longer supported, any `nomodule` scripts should be removed ([#5412](https://github.com/Esri/calcite-components/issues/5412))
+- dropped `dist-custom-elements-bundle` output target – this is deprecated by Stencil and developers should use the `custom-elements` output target instead ([#5411](https://github.com/Esri/calcite-components/issues/5411))
```

Comment:
✨ Very nice! ✨ 

Would we have any sub-header for either of these, perhaps "build" from the previous PRs?

Reply:
> Would we have any sub-header for either of these, perhaps "build" from the previous PRs?

Yeah we could add the `build` scope, so nitpick:

```md
- **build:** dropped ES5 build output – this was disabled since legacy browsers are no longer supported. Any `nomodule` scripts should be removed ([#5412](https://github.com/Esri/calcite-components/issues/5412))
- **build:** dropped [`dist-custom-elements-bundle`](https://stenciljs.com/docs/custom-elements-bundle) output target – this is deprecated by Stencil and developers should use the [`custom-elements`](https://stenciljs.com/docs/custom-elements) output target instead. [Migration instructions](https://developers.arcgis.com/calcite-design-system/get-started/#custom-elements) ([#5411](https://github.com/Esri/calcite-components/issues/5411))
```